src/xrCore/xrCore.h

```diff
@@ -142,6 +142,7 @@ class XRCORE_API xrCore
 public:
     xrCore();
 
+    string64 ApplicationTitle;
```

Comment:
Uhm, sorry that I didn't noticed it earlier:
Logically, `ApplicationTitle` should be lower than `ApplicationName`.

Reply:
```
string64 ApplicationName;
string64 ApplicationTitle;
```
?

Reply:
Yes